about.md

```diff
@@ -6,6 +6,26 @@ title: About MDAnalysis
 <img src="{{ site.baseurl }}/public/mdanalysis-logo_square.png"
 style="float: right" alt="MDAnalysis" width="30%"/>
 
+## About the MDAnalysis Project
+
+The MDAnalysis [Community](#community) is interested in all facets of **working
+with data in the molecular sciences**. We welcome everyone. **We all follow our
+[Code of Conduct] and strive to create an environment that is welcoming to
+all.** Our primary purpose is to produce software that scientists in academia and
+industry will trust to use in their research.
+
+We develop and maintain projects related to the broader goal of processing and
+analyzing data in the molecular sciences. We aim to empower users/developers to
```

Comment:
Here and above "molecular sciences" seems incredibly broad. I suspect someone could propose a molecular science functionality we'd consider out of scope.

That said, I don't really care too much, the general spirit seems correct.

Reply:
That's a fair point... I guess someone could think about NMR data or cryo-ET. 

Are we going to make our life more difficult by keeping it fairly broad? Or does it potentially open up opportunities?

Reply:
In OMSF we often use "computational molecular sciences" to narrow the scope a bit.

Reply:
Sounds good to me. do you want to edit accordingly?

Reply:
Yes please.